doc/configuration.rst

```diff
@@ -2948,6 +2948,48 @@ It supports:
 - connection sharing (DHCP server with NAT)
 - listing DHCP leases (if the client has sufficient permissions)
 
+Binds to:
+  iface:
+    - `NetworkInterface`_
+    - `USBNetworkInterface`_
+    - `RemoteNetworkInterface`_
+
+Implements:
+  - None yet
+
+Arguments:
+  - None
+
+RawNetworkInterfaceDriver
+~~~~~~~~~~~~~~~~~~~~~~~~~
+This driver allows "raw" control of a network interface (such as Ethernet or
+WiFi).
+
+The labgrid-raw-interface helper (``helpers/labgrid-raw-interface``) needs to
+be installed in the PATH and usable via sudo without password.
+A configuration file ``/etc/labgrid/helpers.yaml`` should be installed on hosts
```

Comment:
`s/should/must/` for the stricter check in the helper.

Reply:
Done.